addon/components/session-overview.js

```diff
@@ -316,4 +317,10 @@ export default class SessionOverview extends Component {
 
     yield this.args.session.save();
   }
+
+  @action
+  transitionToCopy() {
+    this.router.transitionTo('session.copy', this.args.session);
```

Comment:
I'm not necessarily opposed to this solution, but it would seem easier to control this from inside the session-copy component itself with `did-insert scroll-into-view`.

Reply:
good point. fixed. please re-review, thanks.